packages/flutter_tools/lib/src/ios/cocoapods.dart

```diff
@@ -150,11 +151,15 @@ class CocoaPods {
   /// Ensures the `ios` sub-project of the Flutter project at [appDirectory]
   /// contains a suitable `Podfile` and that its `Flutter/Xxx.xcconfig` files
   /// include pods configuration.
-  void setupPodfile(String appDirectory) {
+  void setupPodfile(String appDirectory, FlutterManifest manifest) {
     if (!xcodeProjectInterpreter.isInstalled) {
       // Don't do anything for iOS when host platform doesn't support it.
       return;
     }
+    if (manifest.isModule) {
+      // Don't create a Podfile if there is no main project (as for a module).
```

Comment:
main project => host app?
(Is it correct that we'll need to change this, if/when the module gets a materialized host app?)